.processes/community.md

```diff
@@ -0,0 +1,12 @@
+# Community Processes
+
+The HOPR Association _tech_ members, _community_ members and [Project Owner](./development.md#legend) have agreed on the following interaction processes to streamline allocation of _tech_ resources.
+
+## Creating Issues
+
+Community team is expected to create github issues in [this repository](https://github.com/hoprnet/hopr-devrel) using [this template](https://github.com/hoprnet/hopr-devrel/blob/main/.github/ISSUE_TEMPLATE/request-for-tech-work.md).
+
+- issues labelled as `high priority` will be treated as a [file-alarm](./development.md#fire-alarm)
```

Comment:
I also create some analytics task in the same repo. Do we need to distinguish community-tech tasks from them? If so, which label (or other mechanism) should we use?

Reply:
I don't think we need to, basically, the `request for tech work` template can be used by any non-tech HOPR team - most of the tasks created should be prioritized during our epic prio call, if they are marked as `high priority` then we tackle them asap. Same can be for analytics.